rebar.config

```diff
@@ -16,7 +16,7 @@
         {cuttlefish, {git,"git://github.com/kyorai/cuttlefish.git", {branch, "develop"}}},
         {vernemq_dev, {git, "git://github.com/erlio/vernemq_dev.git", {branch, "master"}}},
```

Comment:
maybe we should change the repo for vernemq_dev as well

Reply:
Yeah, I'll do this in another PR